plugins/newsdownloader.koplugin/internaldownloadbackend.lua

```diff
@@ -0,0 +1,56 @@
+local http = require("socket.http")
+local https = require("ssl.https")
+local logger = require("logger")
+local ltn12 = require("ltn12")
+local socket_url = require("socket.url")
+
+local InternalDownloadBackend = {}
+
+local function processDownloadAndReturnSink(url)
```

Comment:
I'd just call it something simple like `get` or `getURL`. Skip the `getResponseAsString` since this should just do `return table.concat(response)`.

Reply:
thanks. I replaced processDownloadAndReturnSink with getResponseAsString  as this ways' better.